course_catalog/etl/loaders.py

```diff
@@ -477,3 +480,54 @@ def load_video_channels(video_channels_data):
         video.save()
 
     return video_channels
+
+
+def load_content_file(course_run, content_file_data):
+    """
+    Sync a course run file/page to the database
+
+    Args:
+        course_run (LearningResourceRun): a LearningResourceRun for a Course
+        content_file_data (dict): File metadata as JSON
+
+    Returns:
+        ContentFile: the object that was created or updated
+    """
+    try:
+        if content_file_data:
+            content_file, _ = ContentFile.objects.update_or_create(
+                run=course_run,
+                key=content_file_data.get("key"),
+                defaults=content_file_data,
+            )
+            return content_file
```

Comment:
That's true of some of the other `load_` functions too (run, program, etc), I thought it would be best to keep this new one consistent even if it's not necessary.